### PR TITLE
Sean l132 water demand manufacturing

### PR DESCRIPTION
**Pull request by swd-turner**

### CODE REVIEW

**Comment by swd-turner** [comment]
@bpbond note that the comparison test data was missing here so I just brought it over from the old system and placed it in _/tests/testthat/comparison_data_ before committing... not sure if that's standard procedure.

**Comment by bpbond** [comment]
@swd-turner re test data - sorry! Perfect, that's all that I do.

File: R/zchunk_L132.water.demand.manufacturing.R [hunk 1]
```diff
@@ -37,40 +37,55 @@ module_water_L132.water.demand.manufacturing_DISABLED <- function(command, ...)
...
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+
+    # Pull out global ratio constants for manufacturing water
+    selfTotalRatio <- manufacturing_water_ratios$`self-to-total-ratio`
+    consWithRatio <- manufacturing_water_ratios$`cons-to-with-ratio`
+
+    # Get total industrial energy use for manufacturing water continent regions for 1995
+    L1322.in_EJ_R_indenergy_F_Yh %>%
+      bind_rows(L1322.in_EJ_R_indfeed_F_Yh) %>%
+      select(GCAM_region_ID, X1995) %>%
```
Comment (bpbond) [inline R/zchunk_L132.water.demand.manufacturing.R]: We'd like to make it easy for ourselves later, when the data isn't coming in through `temp-data-inject` and presumably will be in long form, with no X's in front of years. So please: 1) in line 37 above, do `get_data(...) %>% gather(year, value, ...) %>% mutate(year = as.integer(substr(year, 2, 5)))` and 2) here and below change code appropriate. I.e., so when we match in the real data later, we only have to delete two lines at top of file.

**Comment by bpbond** [comment]
Thanks @swd-turner - looks good. This is waiting on review by @mihejazi

**Pull request merged**